src/style/auto-complete.js

```diff
@@ -0,0 +1,195 @@
+    for selecting the button text
+    */
+    WebkitUserSelect: 'none',
+    MozUserSelect: 'none',
+    MsUserSelect: 'none',
+    userSelect: 'none',
+
+    /* This button can only be pressed */
+    MsTouchAction: 'manipulation',
+    touchAction: 'manipulation',
+
+    /*
+    Prevent flickering while tapping on WebKit
+    http://stackoverflow.com/a/3516243/837709
+    */
+    WebkitTapHighlightColor: 'transparent'
+  },
+
+  hoverStyle: {
+    borderBottom: '1px solid #92D6EF',
+    boxSizing: 'border-box',
+    paddingTop: '7px',
+    paddingBottom: '5px',
+
+    /*
+    To avoid any kind of flickering the user won't get feedback
+    for selecting the button text
+    */
+    WebkitUserSelect: 'none',
+    MozUserSelect: 'none',
+    MsUserSelect: 'none',
+    userSelect: 'none',
+
+    /* This button can only be pressed */
+    MsTouchAction: 'manipulation',
+    touchAction: 'manipulation',
+
+    /*
+    Prevent flickering while tapping on WebKit
+    http://stackoverflow.com/a/3516243/837709
+    */
+    WebkitTapHighlightColor: 'transparent'
+  },
+
+  wrapperStyle: {
+    outline: 0, // to avoid default focus behaviour
+    boxSizing: 'border-box',
+    position: 'relative'
+  },
+
+  menuStyle: {
+    display: 'block',
+    listStyleType: 'none',
+    background: '#FFF',
+    padding: '6px 0',
+    margin: 0,
+    position: 'absolute',
+    width: '100%',
+    zIndex: 10000,
+    boxSizing: 'border-box',
+    borderRadius: 2,
+    boxShadow: '0 1px 1px rgba(0, 0, 0, 0.2)',
+    borderTop: '1px solid #f2f2f2',
+    /* Improve scrolling for mobile Safari */
+    WebkitOverflowScrolling: 'touch'
+  },
+
+  disabledStyle: {
+    borderBottom: '1px dotted #9F9F9F',
+    boxSizing: 'border-box',
+    paddingTop: '7px',
+    paddingBottom: '5px',
+    /*
+    To avoid any kind of flickering the user won't get feedback
+    for selecting the button text
+    */
+    WebkitUserSelect: 'none',
+    MozUserSelect: 'none',
+    MsUserSelect: 'none',
+    userSelect: 'none',
+    /* This button can only be pressed */
+    MsTouchAction: 'manipulation',
+    touchAction: 'manipulation',
+    /*
+    Prevent flickering while tapping on WebKit
+    http://stackoverflow.com/a/3516243/837709
+    */
+    WebkitTapHighlightColor: 'transparent'
+  },
+
+  disabledHoverStyle: {
+    borderBottom: '1px dotted #92D6EF',
+    cursor: 'not-allowed',
+    boxSizing: 'border-box',
+    paddingTop: '7px',
+    paddingBottom: '5px',
+    /*
+    To avoid any kind of flickering the user won't get feedback
+    for selecting the button text
+    */
+    WebkitUserSelect: 'none',
+    MozUserSelect: 'none',
+    MsUserSelect: 'none',
+    userSelect: 'none',
+    /* This button can only be pressed */
+    MsTouchAction: 'manipulation',
+    touchAction: 'manipulation',
+    /*
+    Prevent flickering while tapping on WebKit
+    http://stackoverflow.com/a/3516243/837709
+    */
+    WebkitTapHighlightColor: 'transparent'
+  },
+
```

Comment:
I have extracted out caret image related properties in 3 objects below. The names are similar to the names used for select. But although the select properties style the caret span, these properties style autocomplete input. We can think of some different name for these.
Also right now there is no different hover style for caret, I could not think of one suitable - but might be thats one thing we should add.